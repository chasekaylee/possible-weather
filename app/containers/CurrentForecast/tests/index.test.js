/**
 *
 * Tests for CurrentForecast
 *
 * @see https://github.com/react-boilerplate/react-boilerplate/tree/master/docs/testing
 *
 */

import React from 'react';
import { render } from 'react-testing-library';
// import 'jest-dom/extend-expect'; // add some helpful assertions

import { CurrentForecast } from '../index';

describe('<CurrentForecast />', () => {
  it('Expect to not log errors in console', () => {
    const spy = jest.spyOn(global.console, 'error');
    const dispatch = jest.fn();
    render(<CurrentForecast dispatch={dispatch} />);
    expect(spy).not.toHaveBeenCalled();
  });

  it('Should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(<CurrentForecast />);
    expect(firstChild).toMatchSnapshot();
  });
});
