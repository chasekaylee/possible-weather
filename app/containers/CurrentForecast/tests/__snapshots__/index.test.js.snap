// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CurrentForecast /> Should render and match the snapshot 1`] = `null`;
